### DISCUSSION TITLE
Restarting MediaCMS resets cms/local_settings.py?

### DISCUSSION THREAD

**Discussion by Dan1ell** [mediacms-io/mediacms#76] [Q&A] [answered]
In [the docs](https://github.com/mediacms-io/mediacms/blob/main/docs/Configuration.md) it says:

> A number of options are available on cms/settings.py.
> It is advisable to override any of them by adding it to cms/local_settings.py .
> Any change needs restart of MediaCMS in order to take effect. So edit cms/local_settings.py, make a change and restart MediaCMS

But when I change something in cms/local_settings.py and restart MediaCMS, this seems to reset cms/local_settings.py to its original content. After a while I realized the file was being overwritten with whatever is written in deploy/docker/local_settings.py.
I installed MediaCMS with the Docker Compose installation so I guess that makes sense.

But the text in Configuration.md isn't clear on this point and could be made clearer. The deploy/docker/local_settings.py file is only mentioned in the installation instructions on the main page. It could be mentioned here in Configuration.md as well.

**Comment by mgogoulos** [comment, marked as answer]
good point, I'll edit the Configuration.md page (or if you're willing to do it too feel free to open a PR)

**Reply by Dan1ell** [reply]
Created a PR. 
(I had also made an issue (#85) linked to this discussion, with the goal of linking the PR to the issue.
But turns out I'm not authorized to link PRs to the issues.)
